fastify.js

```diff
@@ -80,7 +80,7 @@ function build (options) {
   var listening = false
   // true when Fastify is ready to go
   var started = false
-  app.on('start', () => {
+  app.on('preReady', () => {
```

Comment:
can you please revert this to `'start'`? Why did you change it?

Reply:
Ooops, a little forgetfulness :P